velox/functions/sparksql/String.cpp

```diff
@@ -146,4 +146,13 @@ std::shared_ptr<exec::VectorFunction> makeLength(
   return kLengthFunction;
 }
 
+void encodeDoubleLengthHexString(uint8_t *data, int size) {
+  static char const HEX_CODES[] = "0123456789abcdef";
```

Comment:
Please, review [coding style](https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md) and update the code to comply.

HEX_CODES -> kHexCodes